receiver/vmmetricsreceiver/metrics_receiver.go

```diff
@@ -107,3 +108,21 @@ func (vmr *Receiver) StopMetricsReception(ctx context.Context) error {
 	})
 	return err
 }
+
+// TODO(songya): investigate why viper.Unmarshal didn't work, remove this method and use viper.Unmarshal instead.
+func unmarshal(cfg *Configuration, settings map[string]interface{}) error {
+	if interval, ok := settings["scrape_interval"]; ok {
+		intervalInSecs := interval.(int)
```

Comment:
this can crash on bad config but assuming that we will figure out this soon and don't need this function anymore

Reply:
Yes, filed https://github.com/census-instrumentation/opencensus-service/issues/522 to keep track.